module/VuFind/src/VuFind/ILS/Driver/VoyagerRestful.php

```diff
@@ -351,11 +351,9 @@ protected function isHoldable($statusArray)
             $valid_hold_statuses_array
                 = explode(':', $this->config['Holds']['valid_hold_statuses']);
 
-            if (!empty($valid_hold_statuses_array)) {
-                foreach ($statusArray as $status) {
-                    if (!in_array($status, $valid_hold_statuses_array)) {
-                        $is_holdable = false;
-                    }
+            foreach ($statusArray as $status) {
```

Comment:
It would make sense to make the `isset()` check above a `!empty()` as well.

Reply:
Done.